Src/AutoFixtureUnitTest/EmailAddressLocalPartTest.cs

```diff
@@ -0,0 +1,169 @@
+﻿using Ploeh.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Extensions;
+
+namespace Ploeh.AutoFixtureUnitTest
+{
+    public class EmailAddressLocalPartTest
+    {
+        [Fact]
+        public void InitializeWithNullLocalPartThrows()
+        {
+            var sut = new EmailAddressLocalPart("good.localPart");
+
+            // Fixture setup
+            // Exercise system and verify outcome
+            Assert.Throws<ArgumentNullException>(
+                () => new EmailAddressLocalPart(null));
+            // Teardown
+        }
+
+        [Theory]
+        [InlineData(@"@CantStartWithAtSign")]
+        [InlineData(@".StartsWithPeriod")]
+        [InlineData(@"Has..TwoPeriods")]
+        [InlineData(@"EndsWithPeriod.")]
+        [InlineData(@".")]
+        [InlineData(@"Ima Notquoted")]
+        public void InitializeWithInvalidLocalPartThrows(string localPart)
+        {
+            // Fixture setup
+            // Exercise system and verify outcome
+            Assert.Throws<ArgumentException>(
+                () => new EmailAddressLocalPart(localPart));
+            // Teardown
+        }
+
+        [Theory]        
+        [InlineData(@"""Quoted\\WithSlash""")]
+        [InlineData("\"Quoted\\\rWithLineBreak\"")]
+        [InlineData(@"""Both\""Quoted""")]
+        [InlineData(@"service/department")]
+        [InlineData(@"$StartWithDollar")]
+        [InlineData(@"!def!xyz%abc")]
+        [InlineData(@"_Good.Email")]
+        [InlineData(@"~")]
+        [InlineData(@"""Quoted@AtSign""")]
+        [InlineData(@"Ima.InternalPeriod")]
+        [InlineData(@"""Ima.Quoted""")]
+        [InlineData(@"""Ima Quoted""")]
+        public void InitializeWithValidLocalPartSetsLocalPart(string localPart)
+        {
+            // Fixture setup            
+            var sut = new EmailAddressLocalPart(localPart);
+            // Exercise system
+            string result = sut.LocalPart;
+            // Verify outcome
+            Assert.Equal(localPart, result);
+            // Teardown
+        }
+
+        [Fact]
+        public void ToStringReturnsCorrectResult()
+        {
+            // Fixture setup
+            string expected = "good.localPart";
+            var sut = new EmailAddressLocalPart("good.localPart");
+            // Exercise system
+            var result = sut.ToString();
```

Comment:
Could you pass this test by implementing `ToString` by always returning the hard-coded string "good.localPart"?